pkg/ocm/managedservice_test.go

```diff
@@ -0,0 +1,33 @@
+package ocm
+
+import (
+	"fmt"
+
+	. "github.com/onsi/ginkgo/v2/dsl/core"
+	. "github.com/onsi/ginkgo/v2/dsl/decorators"
+	. "github.com/onsi/gomega"
+
+	"github.com/openshift/rosa/pkg/fedramp"
+)
+
+var _ = Describe("ManagedService", Ordered, func() {
+	var client Client
+
+	When("FedRAMP is true", func() {
+		It("Can't create managedService", func() {
+			fedramp.Enable() // Simulate logging into fedramp
```

Comment:
@hunterkepley Can you please provide `BeforeEach` and `AfterEach` calls to reset the value of `fedramp.enabled` to false. Because `fedramp.enabled` is a global var, this test has the potential to impact on other test suites by leaving the value as `true`.

```
BeforeEach(func() {
	fedramp.SetEnabled(false)
})
	
AfterEach(func() {
	fedramp.SetEnabled(false)
})
```